pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
Actually it probably is a Go convention. If you `git grep "^package .*_.*"` in the repo you will find many examples.

A black-box test example: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/deep_copy_test.go#L17